install_config/registry/accessing_registry.adoc

```diff
@@ -344,3 +344,50 @@ $ oc adm policy add-cluster-role-to-user prometheus-scraper <username>
 ----
 
 See the link:https://prometheus.io/docs/querying/basics/[upstream Prometheus documentation] for more advanced queries and recommended visualizers.
+
+[[mirror-registry-images]]
+== Transferring Images
+
+To move one or more images from one Docker registry to another,
+use `oc image mirror` command. The images are streamed from registry to registry
+without being stored locally.
+
+For example, to copy image from dockerhub to the integrated registry use the
+following command:
+
+[source, bash]
+----
+$ oc image mirror docker.io/busybox:latest 172.30.124.220:5000/myproject/busybox:latest
+----
+
+It's possible to copy an image to several locations at once. To do this, you
+need to specify several destinations:
+
+[source, bash]
+----
+$ oc image mirror 172.30.124.220:5000/myproject/busybox:latest docker.io/myrepository/busybox:stable docker.io/myrepository/toybox:dev
+----
+
+[NONE]
+====
+The `oc image mirror` runs locally, not inside the openshift cluster. This means
+that this command must have access to the source and destination registry.
+====
+
+If the Docker registry requires authentication to pull or push an image, you
+need to log in manually before executing the command.
```

Comment:
"you need to log in manually (using the `docker login` command) before executing the `oc image mirror` command."